server/index.js

```diff
@@ -0,0 +1,43 @@
+/* eslint consistent-return:0 */
+
+const express = require('express');
+const logger = require('./logger');
+const ngrok = require('ngrok');
+
+const frontend = require('./middlewares/frontendMiddleware');
+const isDev = process.env.NODE_ENV !== 'production';
+
+const app = express();
+
+// If you are developing more than just front-end, like POST form
+// handling or API, add your custom middleware here
+// app.use('/api', myApi);
+
+// Initialize frontend middleware that will server your JS app
+const webpackConfig = isDev
+  ? require('../internals/webpack/webpack.dev.babel')
+  : require('../internals/webpack/webpack.prod.babel');
+
+app.use(frontend(webpackConfig));
```

Comment:
In this case, it probably does not really matter, but wanted to make it webpack independent (so it's location nor the place it's coming from does not mater - likely a candidate for a separate repo / folder - following up our previous discussion of making it agnostic).